shopify/usage-subscription-template/web/components/PlanCard.tsx

```diff
@@ -75,7 +77,7 @@ export default ({
               setDisabled(true);
```

Comment:
Problem in drawer
![CleanShot 2024-11-07 at 15 12 10@2x](https://github.com/user-attachments/assets/c2a0c20d-9b47-48a1-a4ca-150cc9e17a19)

Reply:
I didn't see these errors but I do see that they are of type any. I think that it's an issue that we need to address with the team though